### ISSUE TITLE
Dailies, Update=True problem

### ISSUE THREAD

**Issue by mcrookrumsey** [esoreq/dcarte#19]
I am using dcarte version 0.4.08. 

I am calling:
`
phys = dcarte.load('Physiology_Dailies','PROFILE', update=True)
`

This runs for ~11mins but then get an error: 

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Untitled-1.ipynb Cell 3 in <cell line: 1>()
----> [1](vscode-notebook-cell:Untitled-1.ipynb?jupyter-notebook#W2sdW50aXRsZWQ%3D?line=0) phys = dcarte.load('Physiology_Dailies','PROFILE', reload=True)
      [2](vscode-notebook-cell:Untitled-1.ipynb?jupyter-notebook#W2sdW50aXRsZWQ%3D?line=1) phys = phys.reset_index()

File ~\Anaconda3\envs\sandbox\lib\site-packages\dcarte\utils.py:229, in timer.<locals>.wrapper.<locals>.wrapped(*fun_args, **fun_kwargs)
    227 else:
    228     prefix = f'Finished {desc} in:'
--> 229 out = fun(*fun_args, **fun_kwargs)
    230 elapsed = time.perf_counter() - start
    231 dur = f'{np.round(elapsed,1)}'

File ~\Anaconda3\envs\sandbox\lib\site-packages\dcarte\load.py:60, in load(dataset, domain, **kwargs)
     58     dflt['reapply'] = False
     59     Path(local_file).unlink()
---> 60     return load(dataset,domain,**dflt)
     62 if path_exists(local_file):
     63     if not (dflt['reload'] or dflt['reapply'] or dflt['update']):

File ~\Anaconda3\envs\sandbox\lib\site-packages\dcarte\utils.py:229, in timer.<locals>.wrapper.<locals>.wrapped(*fun_args, **fun_kwargs)
    227 else:
    228     prefix = f'Finished {desc} in:'
--> 229 out = fun(*fun_args, **fun_kwargs)
    230 elapsed = time.perf_counter() - start
...
    695         "Casting to unit-less dtype 'datetime64' is not supported. "
    696         "Pass e.g. 'datetime64[ns]' instead."
    697     )

TypeError: Cannot use .astype to convert from timezone-aware dtype to timezone-naive dtype. Use obj.tz_localize(None) or obj.tz_convert('UTC').tz_localize(None) instead.

```

The same error appears for sleep_dailies and for loading from base
`sleep_dailies = dcarte.load('sleep_dailies','profile', update=True)`
`phys = dcarte.load('Physiology','BASE', update=True)`

**Comment by esoreq** [comment]
This is now fixed in version 0.4.11
Thanks for raising – please confirm so I can close this

From: Mark Crook-Rumsey ***@***.***>
Date: Tuesday, 2 May 2023 at 19:31
To: esoreq/dcarte ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [esoreq/dcarte] Dailies, Update=True problem (Issue #19)
This email from ***@***.*** originates from outside Imperial. Do not click on links and attachments unless you recognise the sender. If you trust the sender, add them to your safe senders list<https://spam.ic.ac.uk/SpamConsole/Senders.aspx> to disable email stamping for this address.



I am using dcarte version 0.4.08.

I am calling:
phys = dcarte.load('Physiology_Dailies','PROFILE', update=True)

This runs for ~11mins but then get an error:

---------------------------------------------------------------------------

TypeError                                 Traceback (most recent call last)

Untitled-1.ipynb Cell 3 in <cell line: 1>()

----> [1](vscode-notebook-cell:Untitled-1.ipynb?jupyter-notebook#W2sdW50aXRsZWQ%3D?line=0) phys = dcarte.load('Physiology_Dailies','PROFILE', reload=True)

      [2](vscode-notebook-cell:Untitled-1.ipynb?jupyter-notebook#W2sdW50aXRsZWQ%3D?line=1) phys = phys.reset_index()



File ~\Anaconda3\envs\sandbox\lib\site-packages\dcarte\utils.py:229, in timer.<locals>.wrapper.<locals>.wrapped(*fun_args, **fun_kwargs)

    227 else:

    228     prefix = f'Finished {desc} in:'

--> 229 out = fun(*fun_args, **fun_kwargs)

    230 elapsed = time.perf_counter() - start

    231 dur = f'{np.round(elapsed,1)}'



File ~\Anaconda3\envs\sandbox\lib\site-packages\dcarte\load.py:60, in load(dataset, domain, **kwargs)

     58     dflt['reapply'] = False

     59     Path(local_file).unlink()

---> 60     return load(dataset,domain,**dflt)

     62 if path_exists(local_file):

     63     if not (dflt['reload'] or dflt['reapply'] or dflt['update']):



File ~\Anaconda3\envs\sandbox\lib\site-packages\dcarte\utils.py:229, in timer.<locals>.wrapper.<locals>.wrapped(*fun_args, **fun_kwargs)

    227 else:

    228     prefix = f'Finished {desc} in:'

--> 229 out = fun(*fun_args, **fun_kwargs)

    230 elapsed = time.perf_counter() - start

...

    695         "Casting to unit-less dtype 'datetime64' is not supported. "

    696         "Pass e.g. 'datetime64[ns]' instead."

    697     )



TypeError: Cannot use .astype to convert from timezone-aware dtype to timezone-naive dtype. Use obj.tz_localize(None) or obj.tz_convert('UTC').tz_localize(None) instead.



The same error appears for sleep_dailies.
sleep_dailies = dcarte.load('sleep_dailies','profile', update=True)

**Comment by mcrookrumsey** [comment]
I have just ran the sleep_dailies update. I can confirm that this works.
